File: read_pdf.py
import sys
import PyPDF2

try:
    with open('d:\\CODE\\revit-mcp\\1.BaiTapMau_Dien.pdf', 'rb') as file:
        reader = PyPDF2.PdfReader(file)
        text = ""
        for page in reader.pages:
            text += page.extract_text() + "\n"
        print(text)
except Exception as e:
    print(f"Error: {e}")

File: RevitMCP_Node/read_pdf.js
import fs from 'fs';
import pdf from 'pdf-parse';

let dataBuffer = fs.readFileSync('d:\\CODE\\revit-mcp\\1.BaiTapMau_Dien.pdf');

pdf(dataBuffer).then(function(data) {
    console.log(data.text);
}).catch(function(error){
    console.error("Error parsing pdf", error);
});
